documentation/movies.workspace.adoc

```diff
@@ -0,0 +1,731 @@
+(PaulBlythe)-[:FOLLOWS]->(AngelaScope)
+
+CREATE
+(JessicaThompson)-[:REVIEWED {summary:'An amazing journey', rating:95}]->(CloudAtlas),
+(JessicaThompson)-[:REVIEWED {summary:'Silly, but fun', rating:65}]->(TheReplacements),
+(JamesThompson)-[:REVIEWED {summary:'The coolest football movie ever', rating:100}]->(TheReplacements),
+(AngelaScope)-[:REVIEWED {summary:'Pretty funny at times', rating:62}]->(TheReplacements),
+(JessicaThompson)-[:REVIEWED {summary:'Dark, but compelling', rating:85}]->(Unforgiven),
+(JessicaThompson)-[:REVIEWED {summary:"Slapstick redeemed only by the Robin Williams and Gene Hackman's stellar performances", rating:45}]->(TheBirdcage),
+(JessicaThompson)-[:REVIEWED {summary:'A solid romp', rating:68}]->(TheDaVinciCode),
+(JamesThompson)-[:REVIEWED {summary:'Fun, but a little far fetched', rating:65}]->(TheDaVinciCode),
+(JessicaThompson)-[:REVIEWED {summary:'You had me at Jerry', rating:92}]->(JerryMaguire)
+
+WITH TomH as a
+MATCH (a)-[:ACTED_IN]->(m)<-[:DIRECTED]-(d) RETURN a,m,d LIMIT 10;
+----
+
+In the next step, you will look for nodes and their properties.
+
+== Find nodes
+
+To start, can you find the actor Tom Hanks?
+
+Run the following query:
+
+[source,cypher]
+----
+MATCH (tom {name: "Tom Hanks"})
+RETURN tom
+----
+
+Next, can you find a movie called _Cloud Atlas_?
+
+[%collapsible]
+.Reveal the solution
+====
+[source,cypher]
+----
+MATCH (movie {title:"Cloud Atlas"})
+RETURN movie
+----
+====
+
+Similarly, you can find all the people (nodes with the `Person` label) in the graph, but limit the returned number to ten:
+
+[source,cypher]
+----
+MATCH (people:Person)
+RETURN people.name LIMIT 10
+----
+
+[TIP]
+====
+Try to remove the property key `.name` from the RETURN clause and see the difference.
+(`RETURN people LIMIT 10`)
+====
+
+If you want to find movies released in a specific timespan, you can add conditions by using `WHERE`:
+
+[source,cypher]
+----
+MATCH (nineties:Movie) WHERE nineties.released >= 1990 AND nineties.released < 2000
+RETURN nineties.title
+----
+
+In the next step you will look for patterns in the graph.
+
+== Find patterns
+
+A pattern in the graph is a specific arrangement of nodes and relationships that can be matched in the graph.
+To find which movies Tom Hanks have acted in, specify the pattern as follows:
+
+[source,cypher]
+----
+MATCH (tom:Person {name: "Tom Hanks"})-[:ACTED_IN]->(tomHanksMovies)
+RETURN tom,tomHanksMovies
+----
+
+Can you specify a pattern to find who directed _Cloud Atlas_?
+Hint: The relationship between a person who is a director and the movie they directed is `:DIRECTED`.
+
+[%collapsible]
+.Reveal the solution
+====
+[source,cypher]
+----
+MATCH (cloudAtlas {title: "Cloud Atlas"})<-[:DIRECTED]-(directors)
+RETURN directors.name
+----
+====
+
+If you extend the length of the pattern, you can find co-actors:
+
+[source,cypher]
+----
+MATCH (tom:Person {name:"Tom Hanks"})-[:ACTED_IN]->(m)<-[:ACTED_IN]-(coActors)
```

Comment:
change `tom` to `person` or better leave it off

Reply:
changed it to `p` for consistency